lib/rubocop/rspec/shared_contexts.rb

```diff
@@ -60,6 +60,67 @@
   end
 end
 
+# This context assumes nothing and defines `cop`, among others.
+RSpec.shared_context 'cop', :cop do # rubocop:disable Metrics/BlockLength
+  ### Meant to be overriden at will
+
+  let(:source) { 'code = {some: :ruby}' }
+
+  let(:processed_source) { parse_source(source, 'test') }
```

Comment:
Do cop tests often touch processed source and the source buffer?

Reply:
Right, I will move `processed_source` in the second section since it is unlikely to be overriden. It is still useful as a shortcust to access the processed source though